doc/reference_processing.xml

```diff
@@ -490,16 +490,16 @@ FROM (SELECT ST_Buffer(
 		<title>Description</title>
 
     <para>
-Clips a geometry by a 2D box in a fast but possibly dirty way. The output
-geometry is not guaranteed to be valid (self-intersections for a polygon
-may be introduced). Topologically invalid input geometries do not result
-in exceptions being thrown.
+Clips a geometry by a 2D box. The output geometry is not guaranteed to be valid
```

Comment:
Is this still accurate?

Reply:
it is not guaranteed but happens to be valid.